### PR TITLE
Added take_an_l writeup

### PR DESCRIPTION
**Pull request by dshynkev**
Since the C code is long here, unlike #1 and #2, this doesn't include the code inline in the `README`. For uniformity's sake, it might make sense to change this on #1 and #2's side.

### CODE REVIEW

File: 2018/csaw_quals/take_an_l/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+`description.pdf` holds further details of the problem.
+One important detail is that the grid is always `64 x 64`
+and we have 5 seconds to send a response.
+
+The solution is simple enough to express recursively:
+- base case: a `2 x 2` square with a mark is tiled in the obvious way
+- inductive step: if a `2^k x 2^k` square with a mark can be tiled,
+  we tile a `2^(k + 1) x 2^(k + 1)` square by splitting it into four
+  `2^k x 2^k` quadrants and tiling an L into quadrants which do not contain
+  a mark already so that each contains exactly one cell comprising the L.
```
Comment (tyxchen) [inline 2018/csaw_quals/take_an_l/README.md]: The wording here is a bit confusing --- just saying "tiling an L with one cell in each quadrant" as in the comments would be clearer

**Review by tyxchen — COMMENTED** [review]
As a reader, I'd definitely prefer to see some diagrams on the recursive process (if possible). Aside from that and the one comment, it looks good to me.

**Comment by dshynkev** [comment]
@tyxchen changed as directed :+1:

**Review by tyxchen — APPROVED** [review]

**Comment by tyxchen** [comment]
lgtm 👍

**Pull request merged**